.github/workflows/release.yml

```diff
@@ -29,6 +30,22 @@ jobs:
           git config --global user.email action@github.com
           git config --global user.name GitHub Action
           echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > .npmrc
+      - name: Setup git SSH remote
+        env:
+          DEPLOY_KEY: ${{ secrets.DEPLOY_KEY }}
+        run: |
+          eval "$(ssh-agent -s)"
+          ssh-add - <<< "${DEPLOY_KEY}"
+          mkdir -p ~/.ssh
```

Comment:
Why are you creating `~/.ssh/id_rsa` after adding the private key to the SSH auth agent?

Reply:
Because Git is not using the SSH Agent somehow 😅

Reply:
Good catch @y-lakhdar, removed the auth agent in 86b20f6